src/pages/app/package/[package].vue

```diff
@@ -31,18 +36,59 @@ async function loadAppInfo() {
       .single()
     app.value = dataApp || app.value
 
-    const date_id = new Date().toISOString().slice(0, 7)
-    const { data } = await supabase
-      .from('app_stats')
+    if (cycleStart && cycleEnd) {
+      const { data } = await supabase
+        .from('stats')
+        .select()
+        .eq('app_id', id.value)
+        .eq('action', 'set')
+        .gte('created_at', getConvertedDate2(cycleStart))
+        .lte('created_at', getConvertedDate2(cycleEnd))
+      if (data)
+        updatesNb.value = data.length
+    }
+    else {
+      const { data } = await supabase
+        .from('stats')
+        .select()
+        .eq('app_id', id.value)
+        .eq('action', 'set')
+      if (data)
+        updatesNb.value = data.length
```

Comment:
same here otherwise some user get wrong numbers:
![CleanShot 2023-09-07 at 15 04 06@2x](https://github.com/Cap-go/capgo/assets/4084527/b220e597-e0f7-411b-a836-c14ade1b2617)